maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/report_old/PluginReport.java

```diff
@@ -218,6 +218,19 @@
                 readonly = true )
     private File enhancedPluginXmlFile;
 
+    /**
+     * In case the internal javadoc site has not been generated when running this report goal
+     * (e.g. when using an aggregator javadoc report) link validation needs to be disabled by setting
+     * this value to {@code true}.
+     * This might have the drawback that some links being generated in the report might be broken
+     * in case not all parameter types and javadoc link references are resolvable through the sites being given to
+     * {@link DescriptorGeneratorMojo}.
+     * 
+     * @since 3.7.0
+     */
+    @Parameter
+    private boolean disableInternalJavadocLinkValidation;
```

Comment:
I would like not to add new parameters to deprecated report - what do you think?

Reply:
A lot of parameters have been added in 3.7.0 (currently both in deprecated and new report goal) not only this one. I don't think that this does any harm here.

Reply:
ok, no problem here